mozilla_version/gecko.py

```diff
@@ -528,6 +593,39 @@ class FirefoxVersion(_VersionWithEdgeCases):
         'build_number': 1,
     })
 
+    _RELEASED_EDGE_CASES_MAJOR = ({
+        'major_number': 1,
+        'minor_number': 5,
+    }, {
+        'major_number': 3,
+        'minor_number': 5,
+    }, {
+        'major_number': 3,
+        'minor_number': 6,
```

Comment:
Version before the introduction of the rapid release scheme didn't have build numbers. Instead, they had release candidates ones.